catalog/spatial/csw/spatial-csw-source/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -35,6 +35,11 @@
                             unbind-method="unbindPlugin"/>
     </reference-list>
 
+    <bean id="cswTransactionWriter" class="org.codice.ddf.spatial.ogc.csw.catalog.common.source.writer.CswTransactionRequestWriter">
```

Comment:
Only the CswCatalogStore implementations should need the CswTransactionWriter - this should be removed.